internal/admin/authorizedapps/save.go

```diff
@@ -86,6 +87,12 @@ func (h *saveController) Execute(c *gin.Context) {
 				m.AddSuccess(fmt.Sprintf("Saved authorized app: %v", authApp.AppPackageName))
 			}
 		}
+
+		if err := addHealthAuthorityInfo(ctx, verdb.New(h.env.Database()), authApp, m); err != nil {
```

Comment:
This puts a _realllll_ tight coupling between three really unrelated components. Is there a way we can decouple these?

Reply:
What's the concern?

authorized apps are tied to health authorities - it's just reflecting the physical model.

Reply:
Just thinking about the problems we've had since injecting the secret manager into database setup.